src/cogent3/app/data_store_new.py

```diff
@@ -502,15 +506,23 @@ def write_log(self, *, unique_id: str, data: str) -> None:
 def get_data_source(data) -> str:
```

Comment:
When I look at this as a user, I get confused about naming. I think other names like `get_source_path` or `get_data_path` or a different name will be more informative.

Reply:
Fair point. The biggest issue is not enough docstrings.